CMakeLists.txt

```diff
@@ -9,7 +9,14 @@ endif()
 
 include(cmake/Policies.cmake)
 
-project(collab-vm-server)
+# example: 3.0.0-0 = Server 3.0.0, plugin interface/protocol version 0? thought this would be useful
+set(COLLABVM_VERSION_MAJOR 3)
+set(COLLABVM_VERSION_MINOR 0)
+set(COLLABVM_VERSION_PATCH 0)
+set(COLLABVM_PINT_VERSION 1)
```

Comment:
Meh, the plugin ABI version is controlled in a .h file and I dunno if I wanna expose that anyways
We will only bump the abi (it's v0 right now) if a huge change is made either way

Reply:
I just wanted to jump on (project VERSION), but we can use them around the cmake files in other ways than just (project VERSION).